public/app/features/dashboard/components/DashNav/DashNavTimeControls.tsx

```diff
@@ -100,6 +100,7 @@ export class DashNavTimeControls extends Component<Props> {
     const timeZone = dashboard.getTimezone();
     const fiscalYearStartMonth = dashboard.fiscalYearStartMonth;
     const hideIntervalPicker = dashboard.panelInEdit?.isEditing;
+    const weekStart = dashboard.weekStart || getWeekStart(config.bootData.user.weekStart);
```

Comment:
Why the use of bootData here? 

what happens to all app plugins that use TimePickerWithHistory and do not set weekStart? 

why have a global in grafana/ui that is named getWeekStart when grafana/runtime has a setWeekStart (that updates locale, which is how we controlled week start before).

This feels like a big breaking change and a very messy change for consistency where all uses of time pickers & calendars need to include this code and use config.bootData (which should mainly for internal use)